src/table/AmpInfo.cc

```diff
@@ -144,23 +144,23 @@ AmpInfoTable::MinimalSchema::MinimalSchema() {
     bboxMin = PointKey<int>::addFields(schema,
         "bbox_min",
         "bbox of amplifier image data on assembled image, min point",
-        "pixels");
+        "pixel");
     bboxExtent = PointKey<int>::addFields(schema,
         "bbox_extent",
         "bbox of amplifier image data on assembled image, extent",
-        "pixels");
+        "pixel");
     gain = schema.addField<double>(
         "gain",
-        "amplifier gain in e-/ADU",
-        "e-/ADU");
+        "amplifier gain",
+        "electron adu^-1");
     saturation = schema.addField<int>(
         "saturation",
-        "saturation value, in ADU",
-        "ADU");
+        "saturation value",
+        "adu");
```

Comment:
As I just posted on the RFC, I don't think we want to use `adu` in some places and `count` in others, since they mean the same thing.  But I'm not sure which is preferred.

(I'm aware this was a preexisting inconsistency, but I'd like to address it on this issue).